Doc/reference/simple_stmts.rst

```diff
@@ -860,7 +860,12 @@ the :ref:`relativeimports` section.
 :func:`importlib.import_module` is provided to support applications that
 determine dynamically the modules to be loaded.
 
-.. audit-event:: import module,filename,sys.path,sys.meta_path,sys.path_hooks import
+.. audit-event:: import module_name,filename,sys.path,sys.meta_path,sys.path_hooks import
+.. audit-event:: import.complete module_name,module import
```

Comment:
This layout ought to be fine. If you were to put an indented (3 space) block of text below, it would replace the generated text completely (and you'd have to list the arguments again).

Worth running the docs build to check the output.

Reply:
I've checked the docs build, and it looks reasonable to me to have clarifying information after the audit-event block.